projects/fal/src/fal/cli/runners.py

```diff
@@ -100,7 +100,10 @@ def _kill(args):
     with FalServerlessClient(
         client._grpc_host, client._credentials
     ).connect() as connection:
-        connection.kill_runner(args.id)
+        if args.force:
+            connection.kill_runner(args.id)
+        else:
+            connection.stop_runner(args.id)
```

Comment:
Lets actually just introduce `fal runners stop`. Similar to how fly does it : there is stop and there is kill.